src/components/common/CourtOrderPoa.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <section id="court-order-poa">
+    <header>
+      <h2>{{sectionNumber}} Court Order and Plan of Arrangement</h2>
+      <div class="py-4">
+        If this filing is pursuant to a court order, enter the court order number. If this
+        filing is pursuant to a plan of arrangement, enter the court order number and select
+        Plan of Arrangement.
+      </div>
+    </header>
+    <div :class="{ 'invalid-section': invalidCourtOrder }">
+      <CourtOrderPoaShared
+        id="court-order"
+        :autoValidation="getAppValidate"
+        :hasDraftPlanOfArrangement="hasDraftPlanOfArrangement"
+        :invalidSection="invalidCourtOrder"
+        @emitCourtNumber="setFileNumber($event)"
+        @emitPoa="setHasPlanOfArrangement($event)"
+        @emitValid="setValidCourtOrder($event)"
+      />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import { Action, Getter } from 'vuex-class'
+import { Component, Prop, Vue } from 'vue-property-decorator'
+import { CourtOrderPoa as CourtOrderPoaShared } from '@bcrs-shared-components/court-order-poa'
+import { ActionBindingIF, FlagsReviewCertifyIF } from '@/interfaces/'
+
+@Component({
+  components: {
+    CourtOrderPoaShared
+  }
+})
+export default class CourtOrderPoa extends Vue {
+  /** Prop to provide section number. */
+  @Prop({ default: '' }) readonly sectionNumber: string
+
+  /** Store getters */
+  @Getter getFlagsReviewCertify!: FlagsReviewCertifyIF
+  @Getter getAppValidate!: boolean
+  @Getter hasDraftPlanOfArrangement!: boolean
```

Comment:
I don't think this Vuex Store getter exists. Did you mean to implement it? Or did you mean to use `getHasPlanOfArrangement` here?

Reply:
I had meant to us `getHasplanOfArrangement` here